interface-impl-amf/src/main/java/org/mule/amf/impl/model/ResourceImpl.java

```diff
@@ -94,36 +92,7 @@ public String getDisplayName() {
 
   @Override
   public Map<String, Parameter> getResolvedUriParameters() {
```

Comment:
why not use this method instead of create a new one ?

Reply:
Because this is part of the model api and I don't want anyone to use this method from outside anymore. It will force to get them through the Action. Does it make sense?

Reply:
why not return uri parameters defined at resource level only ?

Reply:
As we discussed offline, it has not much sense so I moved this to be the default implementation as suggested.